src/core/toga/interface/widgets/button.py

```diff
@@ -101,4 +103,30 @@ def enabled(self, value):
         self._set_enabled(value)
 
     def _set_enabled(self, value):
-        raise NotImplementedError('Button widget must define _set_enabled()')
\ No newline at end of file
+        raise NotImplementedError('Button widget must define _set_enabled()')
+
+    @property
+    def color(self):
+        """
+        Indicates the button color.
+
+        :returns:   Button color. Default is None.
+        :rtype:     ``str` or ``tuple``
+        """
+        return self._color
+
+    @color.setter
+    def color(self, value):
+        """
+        Set the button color.
+
+        :param value:   Button color
+        :type value:    ``str` or ``tuple``
+        """
+
+        self._color = value
+        self._set_background_color(value)
```

Comment:
Why the inconsistency here between `color` and `background_color`? Based on the implementation, this property looks like it should be `background_color`. It would make sense for `color` to be reserved for the text color of the button.